GCMessengerSDKSample/Podfile

```diff
@@ -12,24 +12,12 @@ use_frameworks! :linkage => :static
 install! 'cocoapods',
          :deterministic_uuids => false
 
-DX_CORE="../../../BitBucket/mobiledx-ios-sdk-core/GenesysCloudCore-Dev.podspec"
-DX_MESSENGER="../../../BitBucket/mobiledx-ios-sdk-messenger/GenesysCloudMessenger-Dev.podspec"
-#DX_UI="../../../BitBucket/mobiledx-ios-sdk-ui/GenesysCloud-Dev.podspec"
-
-source 'https://github.com/MyPureCloud/gc-sdk-specs-dev'
+source 'https://github.com/MyPureCloud/gc-sdk-specs'
 source 'https://github.com/CocoaPods/Specs'
 
 target 'GCMessengerSDKSample' do
     pod 'FirebaseCrashlytics'
-
-    if defined? DX_UI
-      pod 'GenesysCloudMessengerTransport', '2.7.2'
-      pod 'GenesysCloudCore', :path => DX_CORE, :testspecs => ['UnitTests']
-      pod 'GenesysCloudMessenger', :path => DX_MESSENGER, :testspecs => ['UnitTests']
-      pod 'GenesysCloud', :path => DX_UI, :testspecs => ['UnitTests', 'ObjcUnitTests']
-    else
-      pod 'GenesysCloud', '1.10.2'
-    end
+    pod 'GenesysCloud', '1.10.2'
```

Comment:
as discussed for this release we will have to lock to 1.10.2 even in main even though it will not be released to gc-sdk-specs during pre_prod
For next release we should remove version lock and add this to the pipeline